libs/ui-watt/.storybook/main.js

```diff
@@ -1,6 +1,15 @@
 module.exports = {
   core: { builder: 'webpack5' },
-
+  // Ideally, the `to` property should be set to 'assets/ui-watt'
+  // so the assets don't have to be placed in a nested "ui-watt" folder.
+  // But due to a path issue on Windows
+  // this is not working properly in Storybook v6.4.x
+  // See https://github.com/storybookjs/storybook/issues/17271
+  //
+  // The bug is fixed in Storybook v6.5.x.
+  // Refactor once we upgrade to v6.5.x
+  // Remember to adjust "app-dh/project.json" file.
```

Comment:
Suggestion: Should we maybe add a "TODO: " here so its easier to remember to find it? :) Or are you just gonna remember? 😄

Reply:
Added a TODO after all :)